src/Components/SceneRefreshConfigurator/SceneRefreshConfigurator.tsx

```diff
@@ -0,0 +1,268 @@
+import React, { useEffect, useState } from 'react';
+import {
+    ISceneRefreshConfiguratorProps,
+    ISceneRefreshConfiguratorStyleProps,
+    ISceneRefreshConfiguratorStyles
+} from './SceneRefreshConfigurator.types';
+import { getStyles } from './SceneRefreshConfigurator.styles';
+import {
+    classNamesFunction,
+    useTheme,
+    styled,
+    Stack,
+    Dropdown,
+    IDropdownOption,
+    IDropdownProps
+} from '@fluentui/react';
+import { useTranslation } from 'react-i18next';
+import HeaderControlButtonWithCallout from '../HeaderControlButtonWithCallout/HeaderControlButtonWithCallout';
+import TooltipCallout from '../TooltipCallout/TooltipCallout';
+import { IPollingStrategy } from '../../Models/Types/Generated/3DScenesConfiguration-v1.0.0';
+import { TFunction } from 'i18next';
+import ViewerConfigUtility from '../../Models/Classes/ViewerConfigUtility';
+import { getDebugLogger } from '../../Models/Services/Utils';
+import { ONE_SECOND, ONE_MINUTE, ONE_HOUR } from '../../Models/Constants';
+
+const debugLogging = false;
+``;
```

Comment:
not sure what this is for, remove please

Reply:
done